docs/platforms/javascript/common/tracing/configure-sampling/index.mdx

```diff
@@ -10,14 +10,14 @@ Sentry's tracing functionality helps you monitor application performance by capt
 
 The JavaScript SDK provides two main options for controlling the sampling rate:
 
-1. Uniform Sample Rate (`tracesSampleRate`)
+### Uniform Sample Rate (`tracesSampleRate`)
 This option sets a fixed percentage of transactions to be captured:
 
 <PlatformContent includePath="/tracing/sample-rate" />
 
 With `tracesSampleRate` set to `0.25`, approximately 25% of transactions will be recorded and sent to Sentry. This provides an even cross-section of transactions regardless of where in your app they occur.
 
-2. Sampling Function (`tracesSampler`)
+### Sampling Function (`tracesSampler`)
```

Comment:
One thing im wondering, based on the feedback from the other Python PR, is if we should actually be changing the language of this section to suggest you can configure "A" traceSampler function; instead of implying this is a function we provide. 

The way it was written originally def implies that its a function we give - which isnt the case if I understand right - but that might also be an unnecessary detail. What do you think?

Reply:
Yeah, that may make sense 🤔 IMHO we can still merge this PR and tweak the exact language we use in a follow up?

Reply:
Yup - im good with that.